extensions/sql-database-projects/src/test/testContext.ts

```diff
@@ -139,3 +139,55 @@ export const mockURIList: vscode.Uri[] = [
 	vscode.Uri.file('/test/folder/folder1/abc1.sqlproj'),
 	vscode.Uri.file('/test/folder/folder2/abc2.sqlproj')
 ];
+
+export const mockConnectionInfo = {
```

Comment:
Can `mockConnectionProfile: azdata.IConnectionProfile` above be used?

Reply:
I am not able to use IConnectionProfile here, because this is of ConnectionInfo object. Even though IConnectionProfile  extends ConnectionInfo, it is not satisfying the test case here. Keeping it as-is for now.

Reply:
Weird.  I wonder if an explicit cast would work...